packages/Table/src/Table.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable react/no-array-index-key */
+import React from "react";
+
+import extractChildren from "@paprika/helpers/lib/extractChildren";
+import PropTypes from "prop-types";
+import ColumnDefinition from "./components/ColumnDefinition";
+import types from "./types";
+import * as sc from "./Table.styles";
+
+const propTypes = {
+  /** Define the look for borders in the table DataGrid.types.GRID, DataGrid.types.NONE, etc.  */
+  borderType: PropTypes.oneOf(["grid", "non", "horizontal", "vertical"]),
+  children: PropTypes.node.isRequired,
+  /** Add an alternate background on the DataGrid's rows */
+  hasZebraStripes: PropTypes.bool,
+  /** Array of data to be stored in the DataGrid */
+  data: PropTypes.arrayOf(PropTypes.shape({})),
+  /** Sets the height of the DataGrid */
+};
+
+const defaultProps = {
+  borderType: types.GRID,
+  data: [],
+  hasZebraStripes: false,
+};
+
+export default function Table(props) {
+  const { borderType, children, hasZebraStripes, data, ...moreProps } = props;
+
+  const { "Table.ColumnDefinition": extractedColumnDefinitions } = extractChildren(children, [
+    "Table.ColumnDefinition",
+  ]);
+
+  let ColumnDefinitions = extractedColumnDefinitions;
+
+  if (!Array.isArray(extractedColumnDefinitions)) {
+    // when there is only one component extracted function return the element not an array of elements
+    ColumnDefinitions = [extractedColumnDefinitions];
+  }
+
+  return (
+    <sc.Table {...moreProps}>
```

Comment:
I feel like we should always provide an `aria-label`. Or else screen reader will just say "2 rows 11 columns" and give no context on what the table is

Reply:
I mean you can totally add it from the consumer side that's why moreProps are there, do you mean we should enforce something like `a11yText` so the consumer always has to provide a label?

Reply:
> According to designs, for the zebra pattern I believe that it should start with white/grey and not grey/white

oh I see good catch will change it.

Reply:
`a11yText` would be nice i think. what do you think?